synth.py

```diff
@@ -27,28 +32,82 @@
 gapic = gcp.GAPICMicrogenerator()
 common_templates = gcp.CommonTemplates()
 
-versions = ['v1', 'v1beta1']
+versions = ["v1", "v1beta1"]
 
 for version in versions:
     library = gapic.typescript_library(
-        'texttospeech',
+        "texttospeech",
         generator_args={
             "grpc-service-config": f"google/cloud/texttospeech/{version}/texttospeech_grpc_service_config.json",
-            "package-name":f"@google-cloud/text-to-speech"
-            },
-        proto_path=f'/google/cloud/texttospeech/{version}',
-        version=version)
+            "package-name": f"@google-cloud/text-to-speech",
+        },
+        proto_path=f"/google/cloud/texttospeech/{version}",
+        version=version,
+    )
 
     # skip index, protos, package.json, and README.md
     s.copy(
-        library,
-        excludes=['package.json', 'README.md', 'src/index.ts'],
+        library, excludes=["package.json", "README.md", "src/index.ts"],
     )
 
-templates = common_templates.node_library(source_location='build/src')
+
+def find_clients(filePath: str) -> typing.List[str]:
+    with open(filePath, "r") as fh:
+        content = fh.read()
+    return re.findall(r"\{(.*Client)\}", content)
```

Comment:
I used regular expression to find the client name because it is generated from `gapic-generator-typescript`. (export {{ '{' + service.name.toPascalCase() + 'Client}' }} from './{{ service.name.toSnakeCase() }}_client';)
I think it is the pattern to extract Client name. @SurferJeffAtGoogle 
@alexander-fenster do you have other way to recommend extract client name from 'version/index.ts'.
Context here, I try to generate a `src/index.ts` here. Then Jefferey will help to locate the function.

Reply:
I would probably look at `^export \{(.*Client)\}`, but this is pretty similar to what you did.
Please verify it works properly when several client are exported, like here: https://github.com/googleapis/nodejs-monitoring/blob/master/src/v3/index.ts